docs/syntax-reference.rst

```diff
@@ -240,30 +238,30 @@ KSQL has currently the following equirements for creating a table from a Kafka t
 
 The WITH clause supports the following properties:
 
-========================= ============================================================================================
-Property                  Description
-========================= ============================================================================================
- KAFKA_TOPIC (required)   | The name of the Kafka topic that backs this table. The topic must already exist in Kafka.
- VALUE_FORMAT (required)  | Specifies the serialization format of message values in the topic. Supported formats:
-                          | ``JSON``, ``DELIMITED`` (comma-separated value), and ``AVRO``.
- KEY (required)           | Associates a field/column within the Kafka message value with the implicit ``ROWKEY`` column
-                          | (message key) in the KSQL table.
-                          |
-                          | KSQL currently requires that the Kafka message key, which will be available as the implicit
-                          | ``ROWKEY`` column in the table, must also be present as a field/column in the message value.
-                          | You must set the KEY property to this corresponding field/column in the message value,
-                          | and this column must be in ``VARCHAR`` aka ``STRING`` format.
-                          | See :ref:`ksql_key_constraints` for more information.
- TIMESTAMP                | By default, the implicit ``ROWTIME`` column is the timestamp of the message in the Kafka
-                          | topic. The TIMESTAMP property can be used to override ``ROWTIME`` with the contents of the
-                          | specified field/column within the Kafka message value (similar to timestamp extractors in
-                          | Kafka's Streams API). Time-based operations such as windowing will process a record
-                          | according to the timestamp in ``ROWTIME``.
-========================= ============================================================================================
++-------------------------+--------------------------------------------------------------------------------------------+
```

Comment:
This table does not render properly in the output (the original design syntax was a bit simpler IMHO, since you don't need to use opening and closing bars (`|`) on each row):
![image](https://user-images.githubusercontent.com/11722533/38328827-c44c8b00-3800-11e8-9e03-2e7aeaa832d4.png)